src/test/java/com/jalasoft/jfc/model/audio/AudioConverterTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.model.audio;
+
+import com.jalasoft.jfc.model.exception.CommandValueException;
+import com.jalasoft.jfc.model.exception.ConvertException;
+import com.jalasoft.jfc.model.exception.ZipJfcException;
+import com.jalasoft.jfc.model.utility.FolderRemover;
+import com.jalasoft.jfc.model.utility.PathJfc;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests AudioConverter class.
+ *
+ * @version 0.1 07 Jan 2020.
+ *
+ * @author Enrique Carrizales.
+ */
+public class AudioConverterTest {
+
+    @Test
+    public void convert_AudioWAVToAudioMP3_Zip() throws ZipJfcException, CommandValueException, ConvertException, IOException {
+        AudioConverter audioConverter = new AudioConverter();
+        AudioParam audioParam = generateOnlyAudioWAV();
+
+        String zipImage = audioConverter.convert(audioParam).getDownload();
+        final int EMPTY_BYTES = 0;
+
+        File zipFile = new File(zipImage);
+        boolean expected = zipFile.exists() && zipFile.getTotalSpace() > EMPTY_BYTES;
+
+        FolderRemover.removeFolder(zipFile.getPath());
+
+        assertTrue(expected);
+    }
+
+    private AudioParam generateOnlyAudioWAV() {
+        PathJfc pathJfc = new PathJfc();
+
```

Comment:
delete this line

Reply:
I add this code line to separate it the other value